Automattic/jetpack
# VideoPress block: modify style for error notice

Fixes 790-gh-Automattic/videopress

#### Changes proposed in this Pull Request:

* Correct alignment and change colors of VideoPress block error message

#### Other information:


#### Jetpack product discussion



Slack discussion:
p1664279922903769/1664277565.111539-slack-C02TQF5VAJD

I used this exchange with design as a guideline:
2847-gh-Automattic/p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Create JN or local test site and install the VideoPress plugin without adding a plan to the site
* Create a post and add a video
* Try to add a second video -> this will return an error because you have no plan
* Confirm that the error looks as expected

How it looked:
<img width="588" alt="Screen Shot 2022-09-28 at 21 52 02" src="https://user-images.githubusercontent.com/16329583/192997202-86eca661-1b90-4f03-83bc-769d06a3210b.png">

How it should look now:
<img width="527" alt="Screen Shot 2022-09-28 at 22 08 52" src="https://user-images.githubusercontent.com/16329583/192997273-abfec03c-da5e-477b-a3fa-608f41f1bfe8.png">Expected
